src/sparse_ir/basis_set.py

```diff
@@ -0,0 +1,40 @@
+from .basis import FiniteTempBasis, finite_temp_bases
+from .sampling import TauSampling, MatsubaraSampling
+
+
+class FiniteTempBasisSet:
+    """Class for holding IR bases and sparse-sampling objects
+
+    An object of this class holds IR bases for fermion and bosons
+    and associated sparse-sampling objects
+    """
```

Comment:
It would be good to add one line for each attribute here:

```suggestion
    Attributes:
        basis_f (FiniteTempBasis): ...
        basis_b (FiniteTempBasis): ...
        ...
    """
```